crates/cairo-profiler/src/trace_reader.rs

```diff
@@ -31,12 +31,12 @@ pub struct Sample {
 }
 
 impl Sample {
-    pub fn extract_measurements(
+    pub fn extract_sample_values(
```

Comment:
This should be performed on the level of reading samples, not on the level of building them.